tests/test_httpsession.py

```diff
@@ -0,0 +1,15 @@
+import botocore
+import pytest
+
+
+async def test_cannot_create_client_sessions_outside_context(session):
+    s3_client_context = session.create_client('s3', 'us-west-2')
```

Comment:
The `session` fixture is different outside the `botocore_tests` scope.

Try passing the following additional arguments to `session.create_client()`

```
        aws_secret_access_key="xxx",
        aws_access_key_id="xxx",
```